adhocracy4/comments/services.py

```diff
@@ -0,0 +1,8 @@
+from .models import Comment
+
+
+def delete_comments(contenttype, pk):
+    comments = Comment.objects.all().filter(
+        content_type=contenttype, object_pk=pk)
+    for comment in comments:
+        comment.delete()
```

Comment:
@xi the code gets all comments for a resource and deletes them, e.g. it is used when an idea is deleted it is used to delete all comments of that idea